research/sentiment_analysis/data/dataset.py

```diff
@@ -0,0 +1,35 @@
+import data.imdb as imdb
+
+DATASET_IMDB = 'imdb'
+
+
+def construct_input_fns(dataset, batch_size, vocabulary_size, sentence_length, repeat=1):
+  """Returns a tuple of input functions, one for training, the other for evaluation.
+  Args:
+    dataset: Dataset to be trained and evaluated. Currently only imdb is supported.
+    batch_size: Number of data in each batch.
+    vocabulary_size: The number of the most frequent tokens to be used from the corpus.
+    sentence_length: The number of words in each sentence. Longer sentences get cut, shorter ones padded.
+    repeat: The number of epoch.
+  Raises:
+    ValueError: if the dataset value is not valid.
+  """
+  if dataset == DATASET_IMDB:
```

Comment:
I think it will be more readable like below, so that dataset does not need to understand each concrete dataset name. 

if dataset == imdb.DATASET_NAME:

Reply:
ack